src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2031,6 +2031,53 @@ limitations under the License.
             <include cluster="Refrigerator And Temperature Controlled Cabinet Mode" client="false" server="false" clientLocked="true" serverLocked="false"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-boolean-sesnsor</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Boolean Sensor</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0041</deviceId>
+        <class>Simple</class>
+        <scope>Endpoint</scope>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="Boolean State" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>TAG_LIST</requireAttribute>
+            </include>
+        </clusters>
+    </deviceType>
+    <deviceType>
+        <name>MA-valve</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Valve</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0042</deviceId>
+        <class>Simple</class>
+        <scope>Endpoint</scope>
+        <clusters lockOthers="true">
```

Comment:
Doesn't this need Descriptor?

Reply:
According to spec, no.

Reply:
Descriptor is mandated by Base Device?

Reply:
Exactly.  _All_ device types inherit from Base Device and hence have Descriptor.

Reply:
Understood. Will fix. I found one more device type that doesn't have Descriptor cluster in it, `MA-smokecoalarm`.

Reply:
https://github.com/project-chip/connectedhomeip/issues/30205